extensions/aliyun-flow/src/constants/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Icon, Image, getPreferenceValues } from "@raycast/api";
+import { Status } from "../types";
+
+export const DOMAIN = "https://openapi-rdc.aliyuncs.com";
+export const ORGANIZATION_ID = getPreferenceValues().organization_id;
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`